catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -134,8 +145,16 @@ protected Object executeWithSubject() throws Exception {
               + "File being imported: {}",
           importFile);
     } else {
-      if (!zipValidator.validateZipFile(importFile)) {
-        throw new CatalogCommandRuntimeException("Signature on zip file is not valid");
+      if (StringUtils.isNotBlank(signatureFile)) {
```

Comment:
✏️ I took a shot at cleaning up this block a bit
```
  if(StringUtils.isBlank(signatureFile)) {
    throw new CatalogCommandRuntimeException("A signature file must be provided with import data");
  }

  String alias = System.getProperty("org.codice.ddf.system.hostname");

  try( FileInputStream fileIs = new FileInputStream(file);
        FileInputStream sigFileIs = new FileInputStream(signatureFile)) {
    if(!verifier.verifyDigitalSignature(fileIs, sigFileIs)) {
      throw new CatalogCommandRuntimeException("The provided data could not be verified");
    }
  }
```
If you do decide to go down the route of changing `verifyDigitalSignature` to accept files instead of input streams

```
  if(StringUtils.isBlank(signatureFile)) {
    throw new CatalogCommandRuntimeException("A signature file must be provided with import data");
  }

  String alias = System.getProperty("org.codice.ddf.system.hostname");
  if(!verifier.verifyDigitalSignature(file, signatureFile, alias)) {
    throw new CatalogCommandRuntimeException("The provided data could not be verified");
  }
```

Reply:
Originally I was passing in `File` objects as parameters however `InputStream` is more flexible because the resource doesn't have to exist in the file system. 

Alternatively I could add two additional methods to `DigitalSignature` that take `File` objects and open input streams.